udf/bits28_to_dates.sql

```diff
@@ -0,0 +1,21 @@
+/*
+
+Convert a bit pattern into an array of the dates is represents.
+
+See detailed docs for the bits28 suite of functions:
+https://docs.telemetry.mozilla.org/cookbooks/normandy_events.html#udf-reference
+
+*/
+CREATE OR REPLACE FUNCTION udf.bits28_to_dates(bits INT64, end_date DATE)
```

Comment:
s/`end_date`/`submission_date`? It could be _not_ `submission_date` if the user shifted the bits, but that seems rare.